.github/dependabot.yml

```
@@ -22,39 +23,29 @@ updates:
           - '@opentelemetry/*' # Update all otel dependencies together
       dev-dependencies: # Update dev dependencies together instead of piecemeal
         patterns:
+          - '@babel*'
+          - '@rollup*'
           - '@type*'
           - 'cypress'
           - 'eslint*'
           - 'jest*'
+          - 'msw'
           - 'prettier'
+          - 'rollup-plugin*'
           - 'ts-*'
+          - 'tslib'
           - 'typescript'
```

```suggestion
        dependency-type: "development"
```

Can we just do this instead or do we need/want to specify the groups with a pattern?

---

Oh yes this is way better, I completely missed this option 🤦🏽‍♀️